src/InnerImageZoom/InnerImageZoom.js

```diff
@@ -9,7 +9,7 @@ class InnerImageZoom extends Component {
     super(props);
 
     this.state = {
-      isActive: false,
+      isActive: (props.startsActive === true ? true : false),
```

Comment:
Would you mind adding `startsActive` to the PropTypes at the bottom of the file? Otherwise this looks good!